README.md

```diff
@@ -1 +1,229 @@
+#Value Type: numeric
+port: 3306
+
+#Description: Wildcard based path to list all audit log files
+#Value Type: string
+audit_log_path: /var/lib/mysql/audit*log*
+
+#Description: List of documented audit admin accounts.
+#Value Type: array
+audit_admins: ["'root'@'localhost'", "'root'@'%'"]
+
+#Description: Name of the documented server cert issuer.
+#Value Type: string
+org_appoved_cert_issuer: CMS Root CA
+
+#Description: List of documented accounts exempted from PKI authentication.
+#Value Type: array
+pki_exception_users: ["healthchecker"]
+
+#Description: List of documented accounts allowed to login with password.
+#Value Type: array
+authorized_password_users: ["healthchecker"]
+
+#Description: List of documented mysql accounts with administrative previlleges.
+#Value Type: array
+mysql_administrative_users: ["root"]
+
+#Description: List of documented mysql administrative role grantees
+#Value Type: array
+mysql_administrative_grantees: ["'root'@'localhost'"]
+
+#Description: List of approved Plugins
+#Value Type: array
+approved_plugins: ["audit_log"]
+
+#Description: List of approved components
+#Value Type: array
+approved_components: ["file://component_validate_password"]
+
+#Description: Authorized MySQL port definitions
+#Value Type: Hash
+mysql_ports:
+  port: 3306
+  admin_port: 33062
+  mysqlx_port: 33060
+
+#Description: Authorized MySQL socket definitions
+#Value Type: Hash
+mysql_sockets:
+  socket: '/var/lib/mysql/mysql.sock'
+  mysqlx_socket: '/var/run/mysqld/mysqlx.sock'
+
+#Description: Location of the my.cnf file
+#Value Type: string
+mycnf: /etc/my.cnf
+
+#Description: Location of the mysqld-auto.cnf file
+#Value Type: string
+mysqld_auto_cnf: /var/lib/mysql/auto.cnf
+
+#Description: Location of the mysqld-auto.cnf file
+#Value Type: array
+authorized_procedures: []
+
+#Description: Location of the mysqld-auto.cnf file
+#Value Type: array
+authorized_functions: []
+
+#Description: Approved minimum version of MySQL
+#Value Type: string
+minimum_mysql_version: 8.0.25
+
+
+```
+
+## Running This Overlay Directly from Github
+
+Against a remote target using ssh (i.e., cinc-auditor installed on a separate runner host)
+```bash
+cinc-auditor exec https://github.com/cms-enterprise/cms-ars-5.0-oracle-mysql-8-stig-overlay/archive/main.tar.gz -t ssh://<target_username>:<target_password>@<target_ip>:<target_port> --input-file <path_to_your_input_file/name_of_your_input_file.yml> --reporter=cli json:<path_to_your_output_file/name_of_your_output_file.json> 
+```
+
+Against a remote target using a pem key (i.e., cinc-auditor installed on a separate runner host)
+```bash
+cinc-auditor exec https://github.com/cms-enterprise/cms-ars-5.0-oracle-mysql-8-stig-overlay/archive/main.tar.gz -t ssh://<target_username>:<target_password>@<target_ip>:<target_port> -i <PEM_KEY> --input-file <path_to_your_input_file/name_of_your_input_file.yml> --reporter=cli json:<path_to_your_output_file/name_of_your_output_file.json>  
```

Comment:
The original command did not have the password placeholder because this option shows the usage of a pem key. @ssayed118